src/main/java/ca/uhn/fhir/jpa/starter/controller/DashboardController.java

```diff
@@ -190,13 +191,14 @@ public ResponseEntity<?> lineChart(
 		}
 		LinkedHashMap<String, String> filters = new LinkedHashMap<>();
 		filters.putAll(allFilters);
-		return helperService.getLineChartByPractitionerRoleId(practitionerRoleId, startDate, endDate, type,filters,env);
+		return helperService.getLineChartByPractitionerRoleId(practitionerRoleId, startDate, endDate, type,filters,env,lga);
```

Comment:
practitionerRoleId are we using this argument in this function getLineChartByPractitionerRoleId, since we are passing the org id directly it is not required. check and remove this argument wherever it is not required